scripts/run-integration-test.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+/* eslint-disable */
+
+const path = require('path');
+const { promises: fs } = require('fs');
+const { promisify } = require('util');
+const { once } = require('events');
+const proc = require('child_process');
+const execFile = promisify(proc.execFile);
+
+const args = process.argv.slice(2);
+
+if (args[0] === 'run-tarball') {
+  run(args.slice(2), args[1]);
+} else if (args[0] === 'pack') {
+  pack(args[1]);
+} else {
+  run(args);
```

Comment:
I would've made `run` the default when `args[0]` is empty, but error out if the command is not recognized. I think this is fine though, given that our commands are wildly different.

Reply:
I made it so that the `run` command is not necessary, because `npm run test:integration run mocha` was ugly. So now it should be `npm run test:integration mocha`. This should be documented somewhere btw. 😅